tools/metrics/grafana/dashboards/buildbuddy.json

```diff
@@ -6457,14 +6380,704 @@
       "type": "row"
     },
     {
-      "collapsed": true,
+      "collapsed": false,
```

Comment:
Grafana saves the collapsed state, refresh interval, etc in the dashboard config so make sure you don't change those.